src/login/logind-core.c

```diff
@@ -411,6 +413,10 @@ int manager_get_idle_hint(Manager *m, dual_timestamp *t) {
 
         assert(m);
 
+        // Initialize the baseline timestamp with the time the manager got initialized to avoid reporting
+        // unreasonable large idle periods starting with the Unix epoch.
```

Comment:
in systemd we use // style comments for "fixme" issues, and /* */ comments for explanatory stuff. This is the latter, hence please use /* */

Reply:
https://systemd.io/CODING_STYLE/#formatting